### DISCUSSION TITLE
Markdown in Messages

### DISCUSSION THREAD

**Discussion by retronbv** [ScratchAddons/ScratchAddons#1703] [Ideas]
What if SA could render markdown in messages? That would be cool right? Normal users would see raw markdown. We could support only italic, strikethrough, bold, underline. Not headings, links, and images. (Should it also render on the messages page & in the popup & in the notifier?)
Just think about it.

**Comment by mxmou** [comment]
This was previously rejected, see #184

**Reply by Explosion-Scratch** [reply]
https://github.com/ScratchAddons/ScratchAddons/issues/184#issuecomment-788069305

**Reply by retronbv** [reply]
@mxmou the issue you linked was not just markdown, honestly I think this would be a very good idea (I might just be a *little* biased)
